app/javascript/components/Results/BestBets/TextBestBet.tsx

```diff
@@ -1,21 +1,32 @@
 import React from 'react';
 import { GridContainer, Grid } from '@trussworks/react-uswds';
 import parse from 'html-react-parser';
+import ResultTitle from '../ResultGrid/ResultTitle';
+import { clickTracking } from '../../../utils';
+import { moduleCode } from '../../../utils/constants';
 
 interface TextBestBetProps {
+  affiliate: string;
   title: string;
   url: string;
   description: string;
+  position: number;
+  query: string;
+  vertical: string;
 }
 
-export const TextBestBet = ({ title, url, description }: TextBestBetProps) => {
+export const TextBestBet = ({ affiliate, title, url, description, position, query, vertical }: TextBestBetProps) => {
   return (
     <GridContainer className='result search-result-item boosted-content'>
       <Grid row gap="md">
         <Grid col={true} className='result-meta-data'>
           <div className='result-title'>
             <h2 className='result-title-label'>
-              <a href={url} className='result-title-link'>{parse(title)}</a>
+              <ResultTitle 
+                url={url}
+                clickTracking={() => clickTracking(affiliate, moduleCode.bestBetsText, query, position, url, vertical)}>
```

Comment:
This can be like
```
<ResultTitle 
 url={result.url}
 className='result-title-link'
clickTracking={() => clickTracking(affiliate, moduleCode.bestBetsText, query, position, url, vertical)}
>
```
Refer implementation at https://github.com/GSA/search-gov/pull/1493/files